frontend/www/js/omegaup/components/course/Edit.vue

```diff
@@ -130,7 +130,10 @@
             (assignmentFormComponent) =>
               $emit('submit-new-assignment', assignmentFormComponent)
           "
-        ></omegaup-course-assignment-details>
+        >
+          <template slot="page-header"><div></div></template>
```

Comment:
```suggestion
          <template slot="page-header"><span></span></template>
```

esto debería evitar que se inserte espacio extra.

Reply:
Listo